### DISCUSSION TITLE
v0.1

### DISCUSSION THREAD

**Discussion by getpinga** [getnamingo/registrar#4] [Announcements]
Attention: We are pleased to announce that all features for the project have been implemented. However, it is not yet ready for production use. We will continue to expand our services gradually, making it easier to work with the billing systems that we support. We welcome any questions or contributions you may have, and we thank you for your patience and support.

<hr /><em>This discussion was created from the release <a href='https://github.com/getpinga/indera/releases/tag/v0.1'>v0.1</a>.</em>

**Comment by DewlanceHosting** [comment]
1. When will it be available for the production use?
2. Does it support the dot IN registry? The IN Registry is the ccTLD for the country of India.

**Comment by getpinga** [comment]
1. It will need at least a few more months for the system to be in production use as the system is focused to become an ICANN registrar software tool.
2. For now if you install Fossbilling.org and https://github.com/getpinga/fossbilling-epp-rfc, you can offer selling of .in domain names even now.
